src/config.ts

```diff
@@ -12,6 +12,23 @@ const AWSX_PROFILE_PATH = path.join(AWSX_HOME, 'profiles');
 const AWS_CREDENTIALS_PATH = path.join(AWS_HOME, 'credentials');
 const AWS_CONFIG_PATH = path.join(AWS_HOME, 'config');
 
+const configFileCheck = (): void => {
+  try {
+    const hasCredentials = fs.existsSync(AWS_CREDENTIALS_PATH);
+    const hasConfig = fs.existsSync(AWS_CONFIG_PATH);
+
+    !hasCredentials && console.error(`You are missing a required file at: ${AWS_CREDENTIALS_PATH}`);
+
+    !hasConfig && console.error(`You are missing a required file at: ${AWS_CONFIG_PATH}`);
+
+    if (!hasCredentials || !hasConfig) {
+      throw Error;
```

Comment:
This needs to be something like `throw new Error('Missing a required config file')`